packages/web-twig/src/Resources/components/Breadcrumbs/index.twig

```diff
@@ -0,0 +1,37 @@
+{# API #}
+{%- set _class = props.class | default(null) -%}
+{%- set _elementType = props.elementType | default('nav') -%}
+{%- set _goBackItemTranslation = props.goBackItemTranslation | default('Back') -%}
+{%- set _items = props.items | default([]) -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Breadcrumbs' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<{{ _elementType }} {{ mainProps(props) }} {{ classProp(_classNames) }} aria-label="Breadcrumb">
+{% apply spaceless %}
+    {%- if items|length > 0 -%}
+        <ol>
+            {% for item in items %}
+                {% if loop.index is same as(items|length - 1) %}
+                    <li class="d-tablet-none">
+                        <a href="{{ item.url }}">{{ _goBackItemTranslation }}</a>
```

Comment:
As I suggested in `we-react`, please invent a better name for a go-back label.

Reply:
Renamed